app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -127,6 +128,16 @@ AccountService = (
         msg = $translate.instant("ERROR.PASSWORD_UPDATE")
       Service.accountError.messages.password = msg
 
+  Service.checkForAccount = (email) ->
+    $http.get("/api/v1/account/check-account?email=#{encodeURIComponent(email)}").success((data) ->
```

Comment:
@chrisdolendo noticed particularly for when using `+` in the email like `dave+test@exygy.com`, it was breaking.